### PR TITLE
Fix or-tools integration

### PR DESCRIPTION
**Pull request by JasonMarechal25**
Update to properly handle new [or-tools-rte repository](https://github.com/rte-france/or-tools-rte)

### CODE REVIEW

File: src/solver/simulation/CMakeLists.txt [hunk 1]
```diff
@@ -45,15 +45,16 @@ set(SRC_SIMULATION
...
+        include/antares/solver/simulation/BindingConstraintsTimeSeriesNumbersWriter.h
+        include/antares/solver/simulation/ISimulationObserver.h
 )
 source_group("simulation" FILES ${SRC_SIMULATION})
 
 #
 # --- Library VARIABLES ---
 #
 add_library(antares-solver-simulation)
+add_library(Antares::antares-solver-simulation ALIAS antares-solver-simulation)
```
Comment (flomnes) [inline src/solver/simulation/CMakeLists.txt:57]:
```suggestion
add_library(Antares::solver-simulation ALIAS antares-solver-simulation)
```

Comment (flomnes) [reply src/solver/simulation/CMakeLists.txt:57]: Also, where is this alias needed, if at all ?

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -265,128 +268,126 @@ OMESSAGE("") # empty line
...
+            set(NSIS_TARGET "Release")
+        else ()
+            set(NSIS_TARGET "Debug")
+        endif ()
+        set(COMPILER_MARK "v")
+        #set(COMPILER_INCLUDE "vs9")
+    else (MSVC)
+        set(NSIS_TARGET "")
+        set(COMPILER_MARK "m")
+        set(COMPILER_INCLUDE "mingw")
```
Comment (flomnes) [inline src/CMakeLists.txt]: If we're going to format CMake files, we need a shared rule for it. Please revert changes related to formatting in this files.

File: src/solver/simulation/CMakeLists.txt [hunk 3]
```diff
@@ -27,7 +27,7 @@ set(SRC_SIMULATION
         include/antares/solver/simulation/solver.hxx
         include/antares/solver/simulation/solver.data.h
         solver.data.cpp
-		include/antares/solver/simulation/simulation-run.h
+        include/antares/solver/simulation/simulation-run.h
```
Comment (flomnes) [inline src/solver/simulation/CMakeLists.txt:30]: You can keep this

File: src/solver/simulation/CMakeLists.txt [hunk 4]
```diff
@@ -45,15 +45,16 @@ set(SRC_SIMULATION
         adequacy_patch_runtime_data.cpp
         include/antares/solver/simulation/ITimeSeriesNumbersWriter.h
         TimeSeriesNumbersWriter.cpp
-		include/antares/solver/simulation/BindingConstraintsTimeSeriesNumbersWriter.h
-		include/antares/solver/simulation/ISimulationObserver.h
+        include/antares/solver/simulation/BindingConstraintsTimeSeriesNumbersWriter.h
+        include/antares/solver/simulation/ISimulationObserver.h
```
Comment (flomnes) [inline src/solver/simulation/CMakeLists.txt:49]: You can keep this

File: src/solver/simulation/CMakeLists.txt (hunk 1 above)
Comment (JasonMarechal25) [reply src/solver/simulation/CMakeLists.txt:57]: https://gopro-collaboratif.rte-france.com/pages/viewpage.action?pageId=411479146
Section: "Gestion des targets dans Simulator "

Comment (JasonMarechal25) [reply src/solver/simulation/CMakeLists.txt:57]: I understand the suggestion but the names need to be the same

**Review by flomnes — APPROVED** [review]

**Pull request merged**